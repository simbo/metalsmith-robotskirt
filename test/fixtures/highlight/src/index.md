
a code block without language definition.

```
var obj = new Shizzle(),
    html = "<p>Foo <strong>Bar</strong></p>";
function foo(bar, baz) {
    return parseInt(bar.val) % baz.getVal();
}
// a comment.
console.log(foo({val: "123"}, obj));
```

a code block with language definition.

``` javascript
var obj = new Shizzle(),
    html = "<p>Foo <strong>Bar</strong></p>";
function foo(bar, baz) {
    return parseInt(bar.val) % baz.getVal();
}

// a comment.
console.log(foo({val: "123"}, obj));
```

a code block with unknown language definition.

``` shizzlemanizzle
var obj = new Shizzle(),
    html = "<p>Foo <strong>Bar</strong></p>";
function foo(bar, baz) {
    return parseInt(bar.val) % baz.getVal();
}

// a comment.
console.log(foo({val: "123"}, obj));
```
