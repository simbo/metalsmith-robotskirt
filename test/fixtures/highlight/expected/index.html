<p>a code block without language definition.</p>
<pre><code><span class="hljs-keyword">var</span> obj = <span class="hljs-keyword">new</span> Shizzle(),
    html = <span class="hljs-string">"&lt;p&gt;Foo &lt;strong&gt;Bar&lt;/strong&gt;&lt;/p&gt;"</span>;
<span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">foo</span><span class="hljs-params">(bar, baz)</span> </span>{
    <span class="hljs-keyword">return</span> <span class="hljs-built_in">parseInt</span>(bar.val) % baz.getVal();
}
<span class="hljs-comment">// a comment.</span>
<span class="hljs-built_in">console</span>.log(foo({val: <span class="hljs-string">"123"</span>}, obj));
</code></pre>
<p>a code block with language definition.</p>
<pre><code class="lang-javascript"><span class="hljs-keyword">var</span> obj = <span class="hljs-keyword">new</span> Shizzle(),
    html = <span class="hljs-string">"&lt;p&gt;Foo &lt;strong&gt;Bar&lt;/strong&gt;&lt;/p&gt;"</span>;
<span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">foo</span><span class="hljs-params">(bar, baz)</span> </span>{
    <span class="hljs-keyword">return</span> <span class="hljs-built_in">parseInt</span>(bar.val) % baz.getVal();
}

<span class="hljs-comment">// a comment.</span>
<span class="hljs-built_in">console</span>.log(foo({val: <span class="hljs-string">"123"</span>}, obj));
</code></pre>
<p>a code block with unknown language definition.</p>
<pre><code><span class="hljs-keyword">var</span> obj = <span class="hljs-keyword">new</span> Shizzle(),
    html = <span class="hljs-string">"&lt;p&gt;Foo &lt;strong&gt;Bar&lt;/strong&gt;&lt;/p&gt;"</span>;
<span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">foo</span><span class="hljs-params">(bar, baz)</span> </span>{
    <span class="hljs-keyword">return</span> <span class="hljs-built_in">parseInt</span>(bar.val) % baz.getVal();
}

<span class="hljs-comment">// a comment.</span>
<span class="hljs-built_in">console</span>.log(foo({val: <span class="hljs-string">"123"</span>}, obj));
</code></pre>